.eslintignore

```diff
@@ -9,6 +9,7 @@ node_modules/
 !.eslintrc.js
 /client/server/devdocs/search-index.js
 /packages/calypso-codemods/tests
+/packages/wpcom.js/webapp/tests/mocha.js
```

Comment:
Not required for this PR, but it will make running eslint (and therefore applying the rule to drop package-relative imports) faster.